pipeline/load.py

```diff
@@ -18,10 +18,10 @@ def get_db_connection() -> connection:
     load_dotenv()
     try:
         return connect(host=environ["DB_HOST"],
-                   user=environ["DB_USER"],
-                   password=environ["DB_PASSWORD"],
-                   port=environ["DB_PORT"],
-                   dbname=environ["DB_NAME"])
+                       user=environ["DB_USER"],
```

Comment:
I havent touched this file idk why it's flagging up

Reply:
Probably a formatting thing on opening and closing the file